core/src/main/java/org/elasticsearch/bootstrap/JarHell.java

```diff
@@ -277,6 +277,9 @@ static void checkClass(Map<String,Path> clazzes, String clazz, Path jarpath) {
                 if (clazz.equals("org.joda.time.base.BaseDateTime")) {
                     return; // apparently this is intentional... clean this up
                 }
+                if (clazz.startsWith("org.apache.logging.log4j.core.impl.ThrowableProxy")) {
+                    return; // deliberate to hack around a bug in Log4j
```

Comment:
can we have a reference to the actual issue in here. It's just easier to reason about this with a direct link?